lib/discourse_antivirus/background_scan.rb

```diff
@@ -28,9 +28,15 @@ def stats
 
     def queue_batch(batch_size: 1000)
       Upload
+        .distinct
```

Comment:
Just curious: why is this needed? You seem to be doing only `left joins` which won't duplicate rows? 🤔

Reply:
`left joins` could still create duplicates if it's a 1:N relation, no? I assumed `upload_references` and `posts` could have multiple references to the same upload.